nemo/collections/tts/helpers/helpers.py

```diff
@@ -206,3 +211,32 @@ def remove(conv_list):
         old_conv = torch.nn.utils.remove_weight_norm(old_conv)
         new_conv_list.append(old_conv)
     return new_conv_list
+
+
+def eval_tts_scores(
+    y_clean: ndarray, y_est: ndarray, T_ys: Sequence[int] = (0,), sampling_rate=22050
+) -> Dict[str, float]:
+    """
+    calculate metric using EvalModule. y can be a batch.
+    Args:
+        y_clean: real audio
+        y_est: estimated audio
+        T_ys: length of the non-zero parts of the histograms
+        sampling_rate: The used Sampling rate.
+
+    Returns:
+        A dictionary mapping scoring systems (string) to numerical scores.
+    """
```

Comment:
Can you make this docstring more descriptive? Specifically mention that you are calculating stoi and pesq.